doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -100,11 +100,96 @@ Thread-based processing
 For better load distribution on the application, enable the experimental thread-based trace processing through the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_THREAD_PROCESSING` Kconfig option.
 Trace processing is done in ``trace_handler_thread``.
 When the modem trace module receives trace data, it places it in a FIFO queue.
-The thread reads from the FIFO queue and forwards the trace data to the configured trace transport medium.
+The thread reads from the FIFO queue and forwards the trace data to the configured trace backend.
 The FIFO queue used by the modem trace module uses dedicated heap memory.
-If the modem trace medium is unable to keep up with the modem traces, the heap size can be increased by enabling :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_HEAP_SIZE_OVERRIDE` and configuring :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_HEAP_SIZE`.
-Increasing the heap size allows more traces in the FIFO queue, but the trace heap will still be depleted if the modem continues to send traces at a rate faster than the rate at which the medium can handle over time.
-If increasing the trace heap size does not help, either optimize the medium speed or use a faster trace transport medium.
+If the modem trace backend is unable to keep up with the modem traces, the heap size can be increased by enabling :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_HEAP_SIZE_OVERRIDE` and configuring :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_HEAP_SIZE`.
+Increasing the heap size allows more traces in the FIFO queue, but the trace heap will still be depleted if the modem continues to send traces at a rate faster than the rate at which the backend can handle over time.
```

Comment:
```suggestion
Increasing the heap size allows more traces in the FIFO queue. However, if the modem continues to send traces at a faster rate than the backend can handle, the trace heap will be exhausted over time.
```

Reply:
Adding a minor suggestion on top of @eivindj-nordic  suggestion.
```suggestion
Increasing the heap size allows more traces in the FIFO queue.
However, if the modem continues to send traces at a faster rate than the backend can handle, the trace heap will be exhausted over time.
```